terminology/README.md

```diff
@@ -0,0 +1,15 @@
+# Terminology
+
+The language we use has power. Being as concise and descriptive as possible helps to lessen ambiguity. This leads to shared understanding and better outcomes.
```

Comment:
This purpose statement is about using more precise technical terms. However, most of the items below are about terms that have problematic racial undertones in a US context. Thoughts on highlighting that here? Otherwise we'll likely end up having discussions about which of two terms is most precise when that's not really the point.

I could see us wanting both types of terminology guidelines here. If so, thoughts on having separate "Non-inclusive terms" and "Ambiguous terms" sections below (we can come up with better header names)? Alternatively, maybe keep both under the same "Terms" section, but add a phrase in parens afterwards explaining why we like one term over the other?

For terms that are merely ambiguous, thoughts on using the weaker verb "Prefer" rather than "Use"?

Reply:
Thematically, I'm not to keen on separating the terminology into "non-inclusive" and "ambiguous" sections, as that does seem counter to the intersectional goals of inclusivity.

I like your idea about adding language specifically as to why. I'll see what I can whip up.

As for ambiguous terms, I'd like to stay with "Use." I view this as a situation where an employee may arrive here looking for guidance, and this is our collective decision to choose one canonical term. Prefer opens up the opportunity for alternate terms, which potentially dilutes shared understanding.